index.html

```diff
@@ -1 +1,130 @@
+          <li><a id="program" href="#Program">Program</a></li>
+          <li><a id="join" href="#Join">Join</a></li>
+          <li><a id="sponsor" href="#Sponsor">Sponsor</a></li>
+          <li class="enroll"><a id="enroll" href="#GetStarted"><strong>Enroll Now</strong></a></li>
+        </ul>
+      </div>
+    </nav>
+  </header>
+<main>
+  <section class="work">
+    <h1 class="title">WEB DEVELOPMENT COURSE 2022</h1>
+    <p class="title-para">Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s.</p>
+    <h2 class="date">24.10.2022 (MONDAY)</h2>
+    <p class="start">Online classes will start on the date mentioned above.</p>
+  </section>
+</main>
+  <section class="highlights">
+    <h3 class="program">Main Program</h3>
+    <hr class="small-hr">
+   <div class="cards-list">
+    <article class="card">
+      <span class="icon">
+        <img src="./Images/html.svg" alt="html">
+      </span>
+      <h2 class="title2">HTML</h2>
+      <p class="title3">Build a strong foundation in web development by learning HTML, one of the major programming languages on the web.</p>
+    </article>
+    <article class="card">
+      <span class="icon">
+        <img src="./Images/css3.svg" alt="css3">
+      </span>
+      <h2 class="title2">CSS</h2>
+      <p class="title3">Build a strong foundation in web development by learning CSS, one of the major programming languages on the web.</p>
+    </article>
+    <article class="card">
+      <span class="icon">
+        <img src="./Images/javascript.svg" alt="javascript">
+      </span>
+      <h2 class="title2">Java<br>Script</h2>
+      <p class="title3">Build a strong foundation in web development by learning JavaScript, one of the major programming languages on the web.</p>
+    </article>
+    <article class="card">
+      <span class="icon">
+        <img src="./Images/react.svg" alt="react">
+      </span>
+      <h2 class="title2">React</h2>
+      <p class="title3">Build a strong foundation in web development by learning React, one of the major programming languages on the web.</p>
+    </article>
+    <article class="card">
+      <span class="icon">
+        <img src="./Images/nodejs.svg" alt="node js">
+      </span>
+      <h2 class="title2">Node</h2>
+      <p class="title3">Build a strong foundation in web development by learning Node Js, one of the major programming languages on the web.</p>
+    </article>
+    </div>
+    <button class="join-btn"><a class="enroll2" href="#">ENROLL IN COURSE 2022</a></button>
+  </section>
+  <section class="speakers">
+    <h3>Featured Instructors</h3>
+    <hr class="fs-hr">
+    <div class="instructors-cards">
+                      </div>
+                      </section>
+  </section>
+  <section class="more">
+    <button class="more-btn"><a class="my-more-btn" href="#">MORE<span><img class="down-img" src="./Images/down.png"></a><span></button>
```

Comment:
Great job @waseemqureshi2020 🤝

- [ ] Kindly make sure this button works as intended. When it is clicked, it takes us to the wrong section instead of displing more speakers.

Reply:
This issue has been fixed.